creativecloud/blocks/cc-forms/cc-forms.css

```diff
@@ -257,6 +257,10 @@
   width: auto;
 }
 
+html[dir="rtl"] .cc-forms .check-item-label {
+  margin-right: var(--spacing-xxs);
+}
+
```

Comment:
If you use `margin-inline-end` you shouldn't have to target `rtl`. Double check.